src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConfig.java

```diff
@@ -63,7 +66,10 @@ private static ConfigDef getConfigDef() {
             .define(WAIT_TIME, Type.LONG, 5000, ConfigDef.Range.atLeast(0), Importance.MEDIUM,
                 "Required, must be zero or higher; the amount of time in milliseconds to wait, on each poll call, " +
                     "before querying MarkLogic. Kafka will continually call poll on the source connector, so this " +
-                    "can be used to control how frequently the connector queries MarkLogic.");
+                    "can be used to control how frequently the connector queries MarkLogic.")
+            .define(KEY_STRATEGY, Type.STRING, "NONE", KEY_STRATEGY_RV, Importance.MEDIUM,
+                "Assigns a key for each record returned; either 'UUID', 'TIMESTAMP', or 'NONE'. If not set key is not returned for the records",
```

Comment:
For the last sentence, say - "If not set, then NONE will be used, in which case a key is not created for each record."